Bixal/move.mil
# Feature 156621182 update slideshow paragraph  w/if-else protips

## Checklist

I have…

      against my changes


## Summary of Changes

This pull request…

- set's logic in the pro-tip template to not display when entity referenced in paragraphs or node renders
- cleaning up some of our linting errors

## Testing

To verify the changes proposed in this pull request…

1. pull branch to local
2. make cim
3. make cr (check to see that it clears "all" without having to confirm [all])
4. cd /move-mil/web/themes/custom/move_mil
5. make build
6. create a Tutorial content, with two paragraphs
-  one with a pro-tip
- one without a pro-tip
note** the one without a pro-tip should not display the pro tip paragraph or styles

## Screenshots

![screencapture-move-mil-localhost-8000-node-4-2018-04-09-14_25_56](https://user-images.githubusercontent.com/4613286/38515377-03d60f80-3c02-11e8-811c-9749b5c98365.png)
